src/lib/crosschainforwarders/CrosschainForwarderArbitrum.sol

```diff
@@ -27,20 +27,34 @@ contract CrosschainForwarderArbitrum {
   uint256 public constant BASE_FEE_MARGIN = 10 gwei;
 
   /**
-   * @dev returns the amount of gas needed for relaying the message based on current basefee
+   * @dev calculateRetryableSubmissionFee on `0x00000000000000000000000000000000000000C8` for a queue call with 1 slot will yield a constant gasLimit of `429478`
+   * To account for some margin we rounded up to 450000
+   */
+  uint256 public constant L2_GAS_LIMIT = 450000;
+
+  /**
+   * @dev There is currently no oracle on L1 exposing gasPrice of arbitrum. Therefore we overpay by assuming 1 gwei (10x of current price).
+   */
+  uint256 public constant L2_MAX_FEE_PER_GAS = 1 gwei;
+
+  /**
+   * @dev returns the amount of gas needed for submitting the ticket
    * @param bytesLength the payload bytes length (usually 580)
    */
-  function getRequiredGas(uint256 bytesLength) public view returns (uint256) {
+  function getMaxSubmissionCost(uint256 bytesLength) public view returns (uint256) {
```

Comment:
For context, why the name change?

Reply:
because required gas was wrong in this context.
In essence the required gas is `maxSubmissionFee + maxRedepmtionFee`, but the method was just returning the first. Anyhow i will refactor to return both as it's probably cleaner.